streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -377,6 +383,18 @@ public boolean process() {
         return true;
     }
 
+    private String getStacktraceString(final KafkaException e) {
```

Comment:
Maybe we could inline this, but my personal preference is to have a separate method call for readability.